### PR TITLE
WIP: g4 changes

### PR DESCRIPTION
**Pull request by tobowers**
These are the changes that are a WIP for getting gossip4 to work

### CODE REVIEW

**Comment by tobowers** [comment]
w00t passing tests

**Comment by tobowers** [comment]
the increase in timeout is that now the wasm is loaded *before* the tests run and that can apparently take more than 2s on the CI server.

**Comment by tobowers** [comment]
moved this to in-review

File: src/tupelo.spec.ts [hunk 1]
```diff
@@ -120,90 +78,60 @@ describe('Tupelo', () => {
...
       })
     return p
-  }).timeout(10000)
-
-  it('gets envelope bits', async () => {
-    const key = await EcdsaKey.generate()
+  })
 
-    const topic = "alongertopicworks"
+  it('gets token payload', async () => {
```
Comment (cap10morgan) [inline src/tupelo.spec.ts]: Every time I read a test like this my brain goes, `it('gets the hose again', ...)` 😶

File: localtupelo/configs/node2/config.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+NotaryGroupConfig = "../localdocker.toml"
+WebsocketPort = 50000
```
[the remark this answers is not in the document]
Comment (tobowers) [reply localtupelo/configs/node2/config.toml]: yeah. 50001 is exposed to the host and bootstrap also needs to have websockets so they share different ports on the host where as node 1 and 2 don't get shared and so they can stay on their 50k

File: src/tupelo.spec.ts (hunk 1 above)
Comment (tobowers) [reply src/tupelo.spec.ts]: https://www.youtube.com/watch?v=20z4edLGxHE

**Pull request merged**